.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
_Note: I couldn't find an open pull request in the frontend app to review.  For your next review request, kindly open one so a proper review can be performed.  It should point from development to main.  I'll add my comments here regarding that app:_

# Frontend app review comments

## On your readme file

- [ ] This note is just a placeholder with instructions for you.  Kindly remove it as it is not intended for your users:

![image](https://user-images.githubusercontent.com/57421823/207945961-eea5051d-5bcf-4c73-a8ac-06f39fe41199.png)

- [ ]  On the "License" section of your readme, the link to the license is broken because it points to a file that does not exist in your project.

### Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

## Other topics

- [ ] Kindly remove the `console.log` debugging code statements present on `addmatch.js, DetailsReserve.js, ReservationForm.js`, in order to improve the repo's cleanliness.

- [ ]  Please, kindly refactor this and all the other functions into ES6 arrow functions, so your code complies with the latest standards.

The `function `keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non-best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/207947811-5087a627-c1b7-430b-a44b-c546733c072b.png)